packages/web/src/hooks/common/use-gnoscan-url.tsx

```diff
@@ -0,0 +1,64 @@
+import { CommonState } from "@states/index";
+import { useAtomValue } from "jotai";
+
+export enum GnoscanDataType {
+  Blocks = "/blocks",
+  Transactions = "/transactions",
+  Realms = "/realms",
+  Tokens = "/tokens",
+  Accounts = "/accounts",
+}
+
+const TEMP_RPC_URL = "https%3A%2F%2Fdev.rpc.gnoswap.io";
+const TEMP_INDEXER_URL = "https%3A%2F%2Findexer-gnoswap.in.onbloc.xyz";
+
+export const useGnoscanUrl = () => {
+  const network = useAtomValue(CommonState.network);
+
+  const getGnoscanUrl = (
+    type: GnoscanDataType | "" = "",
+    params = "",
+  ): string => {
+    const chainId = network.chainId || "";
+    const baseUrl = network.scannerUrl || "";
+    let chainParams = "";
+    if (["portal-loop", "test3"].includes(chainId)) {
```

Comment:
I thought test3 is deprecated

Reply:
yepp that's true. but this check up condition is due to current gnoscan implement.